lib/MetaCPAN/Model/Archive.pm

```diff
@@ -77,7 +77,11 @@ has _tempdir => (
     init_arg => undef,
     lazy     => 1,
     default  => sub {
-        return File::Temp->newdir;
+
+        my $scratch_disk = '/mnt/scratch_disk';
```

Comment:
I misread this.  Should be fine on Travis.  We can fix the paths as part of a cleanup later.